apps/web/src/validators/profile/diversityEquityInclusion.ts

```diff
@@ -12,9 +12,9 @@ export function anyCriteriaSelected({
   indigenousCommunities,
 }: PartialUser): boolean {
   return !!(
-    isWoman ||
-    isVisibleMinority ||
-    hasDisability ||
+    isWoman ??
```

Comment:
I think this should be using ||.

Reply:
Updated in [ignore boolean primitives](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11651/commits/ccb9ba3ec73672fea328bb97fecdb8b1c1db5b91)